Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -370,6 +391,14 @@ - (void)setSelectedDates:(NSArray<BPKSimpleDate *> *)selectedDates {
     self.sameDayRange = selectedDates.count == 2 && [selectedDates.firstObject isEqual:selectedDates.lastObject];
 }
 
+- (BOOL)allowsWholeMonthSelection {
+    return self.selectionConfiguration.wholeMonthTitle != nil && self.selectionConfiguration.wholeMonthTitle.length > 0;
```

Comment:
Should this depend on the kind of configuration that is passed in, rather than checking if a label text is set?

If we want to keep the check like this, It might be nicer to use the wholeMonthTitle that is defined below.

Reply:
In order to simplify it I decided that passing the title for the whole month button should be enough to configure it: no title -> no button.
But I agree with you that it would be nicer to use `wholeMonthTitle` defined below instead of reading it from configuration in both places, so in case of refactoring we only need to change it in one place.